app/lib/efile/az/az140_calculator.rb

```diff
@@ -208,7 +230,7 @@ def calculate_line_52
       end
 
       def calculate_line_53
-        # zzz
+        # state taxes withheld: sum of W-2, 1099-R, 1099-G
```

Comment:
should 1099R be added in this calculation below?

Reply:
after chatting with tiffany and courtney I learned that there are some more 1099R fields we should add, including the state taxes withheld, so I'm going to add that into this PR ([slack thread here](https://cfastaff.slack.com/archives/C07GD2NS086/p1724789015247049))